requirements.tex

```diff
@@ -24,10 +29,17 @@ \subsection{Release Requirements for the \gls{LSST} Science Community} \label{se
 The \gls{DM} Science Team will liaise with the \gls{LSST} science community to understand their needs concerning \gls{DM} software product releases. 
 Requests for new functionality or back-ported bug fixes to include in forthcoming \gls{DM} software product releases will be communicated to \gls{DM} via a \gls{RFC}, coordinated either by the DM-SST liaison to the science collaboration or a nominated person within the science collaboration who will coordinate requests and raise an \gls{RFC}. 
 
+
 \subsection{Release Requirements for Data Processing in Operations} \label{sec:procreqs}
 
-The Science Operations Department will be in charge of  data processing in operations. 
-Software for the various pipelines will be hosted at \gls{NCSA},IN2P3 and in Chile.
+%The Science Operations Department will be in charge of  data processing in operations. 
+%Software for the various pipelines will be hosted at \gls{NCSA},IN2P3 and in Chile.
+
+The \gls{LDF} will be responsible for generating the needed data products during commissioning and operations. 
```

Comment:
The \gls{LDF} will be responsible for generating the necessary data products during commissioning and operations.